.github/actions/prlinter/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "prlinter",
```

Comment:
`private`?

Reply:
Mmm not sure I follow. What do you mean? `prlinter` --> `PullRequestLinter`

Reply:
I mean the package should be marked private so it does not get pushed to `npmjs` by mistake.

Reply:
Got it, thanks! good catch.